miniDAS/format.py

```diff
@@ -0,0 +1,161 @@
+from __future__ import annotations
+
+import logging
+from dataclasses import asdict, dataclass, field
+from pathlib import Path
+from typing import Any, Literal
+
+import h5py
+import numpy as np
+
+DataUnit = Literal["m/m", "cm/m", "mm/m"]
+ScaledUnit = Literal["ue/m"]
+_ALLOWED_DTYPES = (np.int32, np.float32)
+
+
+class OutOfBoundsError(BaseException):
+    ...
+
+
+@dataclass
+class Meta:
+    data_unit: DataUnit
```

Comment:
In an earlier comment you suggested to include Vendor: String, and Interogator_Model: String
I would offer that as optional strings both can be empty strings ""